pom.xml

```diff
@@ -34,7 +34,16 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
-
+        <dependency>
+            <groupId>org.springframework.data</groupId>
+            <artifactId>spring-data-jpa</artifactId>
```

Comment:
Deze moet je niet gebruiken. Wij gebruiket de spring-boot-starter-data-jpa. Dat is een starter dependency van SpringBoot, dit is een dependency van Spring. Deze twee doen bijna hetzelfde, maar zijn bedoeld voor net iets andere situaties. Daarom is er grote kans dat ze conflict met elkaar kunnen geven waardoor je rare errors krijgt in je applicatie.